subsys/usb/class/audio/Kconfig

```diff
@@ -0,0 +1,107 @@
+# USB AUDIO configuration options
+
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config USB_DEVICE_AUDIO
+	bool "USB Audio Device Class Driver"
+	help
+	  USB audio device class driver
+
+if USB_DEVICE_AUDIO
+
+config USB_AUDIO_DEVICE_NAME
+	string "USB AUDIO device name template"
+	default "AUDIO"
+	help
+	  Device name template for the AUDIO Devices. First device would have
+	  name $(USB_AUDIO_DEVICE_NAME)_0, etc.
+
+module = USB_AUDIO
+module-str = USB Audio
+source "subsys/logging/Kconfig.template.log_config"
+
+choice
+	prompt "Device type"
+menuconfig USB_AUDIO_HEADPHONES
+	bool "Headphones"
+	help
+	  USB audio headphones config options
+if USB_AUDIO_HEADPHONES
+menuconfig USB_AUDIO_HEADPHONES_FEATURES
+	bool "Headphones features"
+	default y
+	help
+	  Choose features that headphones supports
+if USB_AUDIO_HEADPHONES_FEATURES
+config USB_AUDIO_HEADPHONES_FEATURES_MUTE
+	bool "Mute"
+	default y
+	help
+	  Enable Mute feature
+config USB_AUDIO_HEADPHONES_FEATURES_VOLUME
+	bool "Volume"
+	default y
+	help
+	  Enable Volume feature
+config USB_AUDIO_HEADPHONES_FEATURES_TONE_CONTROL
+	bool "Tonce Control"
+	help
+	  Enable Tone Control (Bass, Mid, Treble) feature
+config USB_AUDIO_HEADPHONES_FEATURES_GRAPHIC_EQUALIZER
+	bool "Graphic Equalizer"
+	help
+	  Enable Graphic Equalizer feature
+config USB_AUDIO_HEADPHONES_FEATURES_AUTOMATIC_GAIN_CONTROL
+	bool "Automatic Gain Control"
+	help
+	  Enable Automatic Gain Control feature
+config USB_AUDIO_HEADPHONES_FEATURES_DELAY
+	bool "Delay"
+	help
+	  Enable Delay feature
+config USB_AUDIO_HEADPHONES_FEATURES_BASS_BOOST
+	bool "Bass Boost"
+	help
+	  Enable Bass Boost feature
+config USB_AUDIO_HEADPHONES_FEATURES_LOUDNESS
+	bool "Loudness"
+	help
+	  Enable Loudness feature
+endif #USB_AUDIO_HEADPHONES_FEATURES
+
+choice
+	prompt "Data format"
+
+config DATA_FORAMT_48KHZ
```

Comment:
```suggestion
config USB_AUDIO_DATA_FORMAT_48KHZ
```

Reply:
When it comes to this. I added this option just for testing purposes. I don't even know if this is relevant for audio.